server/nuxeo-launcher/src/test/java/org/nuxeo/launcher/TestNuxeoLauncher.java

```diff
@@ -172,4 +175,27 @@ private void checkParsing(NuxeoLauncher launcher) throws GeneralSecurityExceptio
         launcher.encrypt();
     }
 
+    /*
+     * NXP-29401
+     */
```

Comment:
if it's a simple comment shouldn't it be one-lined?

Reply:
I would say it is free. As I like to have room in the code for clarity, I converted it to javadoc.

Reply:
but shouldn't it start with two stars to be considered Javadoc?

Reply:
You can have a look at the code which reflects the change https://github.com/nuxeo/nuxeo/pull/4281/files#diff-c7467595cf49ff4ffa220ef476aa4384R178

Reply:
oh sorry, outdated!